src/main/java/jeff/UserInterface.java

```diff
@@ -0,0 +1,53 @@
+package jeff;
+
+import jeff.exceptions.*;
+import java.util.Scanner;
+
+public class UserInterface {
+    public static void handleUserInput() {
```

Comment:
method abstraction is very well done, however it is a little longer than what would be acceptable (>30 LOC) see if you can abstract something out into a different function 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods